src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine machineInSector1 = new Bulldozer();
+        Machine machineInSector2 = new Excavator();
+        Machine machineInSector3 = new Truck();
+        machineInSector1.doWork();
+        machineInSector2.doWork();
+        machineInSector3.doWork();
+        machineInSector1.stopWork();
+        machineInSector2.stopWork();
+        machineInSector3.stopWork();
```

Comment:
let's create a `Machine` array. After that you can go through this array, using for loop and call methods `doWork()` and `doStop()` in it.